tigerpath/majors_and_certificates/degrees/AB_2018.json

```diff
@@ -31,8 +57,7 @@
       "name": "Writing Seminar",
       "max_counted": 1,
       "min_needed": 1,
-      "description": "Writing Seminar — one course",
-      "explanation": "All Princeton undergraduates take a Writing Seminar in the Fall or Spring of the freshman year.\n\nThe Writing Seminars give Princeton freshmen an early opportunity to belong to a lively academic community in which members investigate a shared topic and discuss their writing together, with the aim of clarifying and deepening their thinking. The seminars are interdisciplinary in nature to emphasize transferable reading, writing and research skills. You’ll learn how to frame compelling questions, position your argument within a genuine academic debate, substantiate and organize claims, purposefully integrate a wide variety of sources and revise for greater cogency and clarity.\n\nEnrollment process:\nhttps://writing.princeton.edu/seminars/how-enroll",
+      "explanation": "<b>Writing Seminar</b><br><i>one course, completed in the first year</i><hr>All Princeton undergraduates take a Writing Seminar in the Fall or Spring of the freshman year.\n\nThe Writing Seminars give Princeton freshmen an early opportunity to belong to a lively academic community in which members investigate a shared topic and discuss their writing together, with the aim of clarifying and deepening their thinking. The seminars are interdisciplinary in nature to emphasize transferable reading, writing and research skills. You’ll learn how to frame compelling questions, position your argument within a genuine academic debate, substantiate and organize claims, purposefully integrate a wide variety of sources and revise for greater cogency and clarity.\n\nMore information:\n<a href=\"https://writing.princeton.edu/undergraduates/writing-seminars/fall-seminars\" target=\"_blank\" rel=\"noopener\">https://writing.princeton.edu/undergraduates/writing-seminars/fall-seminars</a>\n\nEnrollment process:\n<a href=\"https://writing.princeton.edu/seminars/how-enroll\" target=\"_blank\" rel=\"noopener\">https://writing.princeton.edu/seminars/how-enroll</a>",
```

Comment:
That is some ghastly looking code...

Reply:
What do you suggest?

Reply:
Chatted with Barak - we decided to leave it for now